### PR TITLE
Removes references to `normalized_score` since it's not a thing now

### PR DESCRIPTION
**Pull request by awlevin**
as stated :)

### CODE REVIEW

File: fern/docs/content/help/metrics/custom-metrics.mdx [hunk 1]
```diff
@@ -133,9 +132,8 @@ The Workflow Metric allows you to use a Workflow to evaluate outputs, allowing L
 You'll map these to the Test Suite variables when setting up the Metric later, so you can name them anything you want.
 Examples of variables you may want to include: the output to be evaluated,  the desired output, the inputs to the evaluated prompt.
 3. Create a Final Output, set the name to `score`, and set the output type to Number.
-4. [Optional] Create a Final Output, set the name to `normalized_score`, and set the output type to Number. If included, `normalized_score` must be in the interval [0, 1]. This value is used for display purposes such as color coding outputs.
```
Comment (noanflaherty) [inline fern/docs/content/help/metrics/custom-metrics.mdx:136]: Nice to have: we could mention here that you can optionally include other outputs such as “rationale”

**Review by noanflaherty — APPROVED** [review]
🔥

File: fern/docs/content/help/metrics/custom-metrics.mdx (hunk 1 above)
Comment (awlevin) [reply fern/docs/content/help/metrics/custom-metrics.mdx:136]: Great call

Comment (awlevin) [reply fern/docs/content/help/metrics/custom-metrics.mdx:136]: @noanflaherty made the change, shipping now, lmk post-ship if you want any more tweaks on it

**Pull request merged**